backend/tests/conftest.py

```
@@ -0,0 +1,137 @@
+async def client() -> AsyncClient:
+    async with AsyncClient(
+        transport=ASGITransport(app=app),
+        base_url="http://test",
+    ) as ac:
+        yield ac
+
+
+async def _signup_and_verify_volunteer(client: AsyncClient) -> tuple[dict, AsyncClient]:
+    """봉사자 회원가입 → 이메일 인증 → 로그인된 클라이언트 반환"""
+    email = unique_email("vol")
+    password = "TestPass1234!"
```

Ruff (0.15.13)

[error] Possible hardcoded password assigned to: "password"

(S105)